### ISSUE TITLE
[v3] Add prerequisites 3/6

### ISSUE THREAD

**Issue by TomPradat** [exercism/javascript#984]
```
anagram

acronym

high-scores

isogram

matching-brackets

phone-number

scale-generator

series

largest-series-product

transpose

grep

rectangles

spiral-matrix

ocr-numbers

saddle-points

forth

food-chain

house

isbn-verifier

pig-latin
```

**Comment by AnishDe12020** [comment]
I am willing to take this issue up (at least some of the exercises). Just let me know if I should make the PRs, one for each exercises, one for a batch of exercises or one for all of them.

**Comment by TomPradat** [comment]
I'd suggest that you make a PR for each exercise but if you feel like you want to do a lot of exercises you can take 4-5 by PR.

Trying to do all of them at once is not a good idea IMO, it's indigestible for both you and the reviewer.

Idk if this might help but here is the steps I used when doing some :

1 - Doing/trying the exercise
2 - Looking at the exercise's proof of work
3 - Looking at the submitted solutions.
4 - Adding the prerequisites

**Comment by AnishDe12020** [comment]
> I'd suggest that you make a PR for each exercise but if you feel like you want to do a lot of exercises you can take 4-5 by PR.
> 
> Trying to do all of them at once is not a good idea IMO, it's indigestible for both you and the reviewer.
> 
> Idk if this might help but here are the steps I used when doing some :
> 
> 1 - Doing/trying the exercise 2 - Looking at the exercise's proof of work 3 - Looking at the submitted solutions. 4 - Adding the prerequisites

Sounds good to me

**Comment by TomPradat** [comment]
Here is an updated list (25/10/2022) :

```
matching-brackets

scale-generator

series

largest-series-product

transpose

grep

rectangles

spiral-matrix

ocr-numbers

saddle-points

forth

food-chain

house

isbn-verifier

pig-latin

```

**Comment by safwansamsudeen** [comment]
Is this still open? Would love to work on it if it is!

**Comment by TomPradat** [comment]
@safwansamsudeen Yes ! See https://github.com/exercism/javascript/issues/984#issuecomment-984682274 for suggestions on how to do it. Let me know if you have any questions

**Comment by junedev** [comment]
@safwansamsudeen If you work on this, please start out with a small batch, e.g. 3 exercises max.
Add a GitHub comment for each one explaining which solution you had in mind when selecting the concepts you listed. Maybe paste a link to a community solution or paste some code. This is the only way the PR becomes reviewable. Without this, the reviewer needs to do the exact work you did to come up with the list again to verify it makes sense.

**Comment by junedev** [comment]
@TomPradat Please keep this in mind https://exercism.org/blog/freeing-our-maintainers and that neither JD not me are currently actively maintaining this repo. Only encourage community contributions if you are confident that you have the time to properly review them when they come in. Otherwise it will be a frustrating experience for everyone.

**Comment by safwansamsudeen** [comment]
@junedev I'm new to both OSS and Exercism, so I think I'll start off with one exercise per PR. 

@TomPradat, I've currently opened a PR - #2159 - for the [House](https://exercism.org/tracks/javascript/exercises/house) exercise. Let me know how it goes!

**Comment by TomPradat** [comment]
@safwansamsudeen I'll have a look next week when I have a bit more time 👍

**Comment by quintuple-mallard** [comment]
Is this still available? I'd really like to work on it if it is, it seems like a good first contribution.
